cni/network/network.go

```diff
@@ -999,6 +1052,7 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 			plugin.ipamInvoker = NewCNSInvoker(k8sPodName, k8sNamespace, cnsClient, util.ExecutionMode(nwCfg.ExecutionMode), util.IpamMode(nwCfg.IPAM.Mode))
 
 		default:
+			// nwInfo gets populated later in the function
 			plugin.ipamInvoker = NewAzureIpamInvoker(plugin, &nwInfo)
```

Comment:
how this work for dualnic scenario?

Reply:
in dualnic , endpoints belong to 2 different networks

Reply:
assuming stateful, we call getEndpointInfosFromContainerID, which loops over all networks and finds the endpointInfos that match the container id. Then we call Delete Endpoint for each one.